packages/core/content-releases/admin/src/pluginId.ts

```diff
@@ -0,0 +1,3 @@
+import pluginPkg from '../../package.json';
+
+export const pluginId = pluginPkg.name.replace(/^@strapi\//i, '');
```

Comment:
do you actually need this? Couldn't you just export the string value? 😄

Reply:
Maybe right now it is used just in two files for the content releases, but in the future can be helpful in other places to avoid typos when I need the plugin name, in addiction I saw that is broadly used in the repo, I don't think it is a big deal to have it also in the content releases tbh

Reply:
yeah im not saying get rid of it, but cant you just right the strong instead of this over-engineered regex?

Reply:
ah ok, sure